### PR TITLE
listing compulsory before get_check()

### PR DESCRIPTION
**Pull request by maelle**
cf #260 where I find the current error behavior puzzling.

### CODE REVIEW

File: R/check-class.R [hunk 1]
```diff
@@ -392,6 +392,14 @@ first_line <- function(x) {
...
   assert_that(is_check_ids(ids))
 
+  
+  if (length(package_data$ids) == 0 && 
+      length(package_data$groups) == 0){
+    stop(paste0("No check IDs in cache, please run list_my_checks()",
+                " or list_package_checks() first."),
+         call. = FALSE)
+  }
+
```
Comment (gaborcsardi) [inline R/check-class.R]: You don't need to `paste0()`, `stop()` concatenates by default:
```r
❯ stop("foo", "bar")
Error: foobar
```

**Comment by maelle** [comment]
@gaborcsardi other comments?

**Comment by gaborcsardi** [comment]
Hmmm. It actually should work if someone specifies the full id of the check. But I have know idea what happens in this case.

**Comment by maelle** [comment]
So the error message should only happen if the id given by the user has too few characters?

**Comment by gaborcsardi** [comment]
Thanks!

**Pull request merged**